File: src/main/java/com/icloud/visitor/after/Phone.java
package com.icloud.visitor.after;

public class Phone implements Device {

    @Override
    public void print(Circle circle) {
        System.out.println("Print Circle to Phone");
    }

    @Override
    public void print(Rectangle rectangle) {
        System.out.println("Print Rectable to Phone");
    }

    @Override
    public void print(Triangle triangle) {
        System.out.println("Print Trigangle to Phone");
    }
}

File: src/main/java/com/icloud/visitor/review/device/Phone.kt
package com.icloud.visitor.review.device

import com.icloud.visitor.review.shape.Circle
import com.icloud.visitor.review.shape.Rectangle
import com.icloud.visitor.review.shape.Star

class Phone : Device {
    override fun print(shape: Circle) {
        println("Print Circle to Phone")
    }

    override fun print(shape: Rectangle) {
        println("Print Rectangle to Phone")
    }

    override fun print(shape: Star) {
        println("Print Start to Phone")
    }

}